FruitFruit/FruitFruit/Extension/NSMutableAttributedString_Extension.swift

```diff
@@ -8,10 +8,10 @@
 import UIKit
 
 extension NSMutableAttributedString {
-        func setColorForText(textToFind: String, withColor color: UIColor) {
-         let range: NSRange = self.mutableString.range(of: textToFind, options: .caseInsensitive)
-          if range != nil {
-              self.addAttribute(NSAttributedString.Key.foregroundColor, value: color, range: range)
-          }
+    func setColorForText(textToFind: String, withColor color: UIColor) {
+        let range: NSRange = self.mutableString.range(of: textToFind, options: .caseInsensitive)
+        if range != nil {
+            self.addAttribute(NSAttributedString.Key.foregroundColor, value: color, range: range)
         }
```

Comment:
```suggestion
        guard let range = self.mutableString.range(of: textToFind, options: .caseInsensitive) else { return }
        self.addAttribute(NSAttributedString.Key.foregroundColor, value: color, range: range)
```

guard let을 확인하면 optional 여부를 확인할 필요가 없어질 듯해요